src/Pages/LandingPage/Component/BookCard/Tiket.jsx

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import styled from 'styled-components';
+import { flexCenter, theme } from '../../../../Styles/CommonStyle';
+
+const Tiket = ({
+  colorTitle,
+  title,
+  subscription,
+  firstText,
+  secondText,
+  monthPrice,
+  yearPrice,
+  totalPrice,
+  firstComment,
+  secondComment,
+  backgroundColor,
+  fontColor,
+  firstBoxColor,
+  secondBoxColor,
+}) => {
+  const MP = parseInt(monthPrice);
+  const YP = parseInt(yearPrice);
+  const TP = parseInt(totalPrice);
```

Comment:
숫자형으로 바꾸고 싶으신 걸까요? parseInt는 성능 이슈가 있습니다. Number()나 +를 앞에 붙여주세요
```suggestion
  const MP = +monthPrice;
  const YP = +yearPrice;
  const TP = +totalPrice;
```

Reply:
대박 이런게 있는줄 몰랐습니다!!! 감사합니다!!